doc/rds.md

```diff
@@ -0,0 +1,31 @@
+# Amazon RDS
```

Comment:
Let's just add a quick sentence before this line, carifying that gh-ost actually runs on RDS and Aurora.